typescript/src/accounts/evm/resolveViemClients.ts

```diff
@@ -0,0 +1,209 @@
+import {
+  Address,
+  createPublicClient,
+  createWalletClient,
+  http,
+  type Chain,
+  type PublicClient,
+  type WalletClient,
+} from "viem";
+import { base, baseSepolia, mainnet, sepolia } from "viem/chains";
+
+/**
+ * Network identifier to viem chain mapping
+ */
+const NETWORK_TO_CHAIN_MAP: Record<string, Chain> = {
+  base: base,
+  "base-sepolia": baseSepolia,
+  mainnet: mainnet,
+  ethereum: mainnet,
+  sepolia: sepolia,
+  eth: mainnet,
+};
+
+/**
+ * Chain ID to viem chain mapping
+ */
+const CHAIN_ID_TO_CHAIN_MAP: Record<number, Chain> = {
```

Comment:
I think we should use the `getChain` helper I mentioned:
```ts
import * as chains from "viem/chains";

/**
 * Get a chain from the viem chains object
 *
 * @param id - The chain ID
 * @returns The chain
 */
export const getChain = (id: string): Chain => {
  const chainList = Object.values(chains);
  return chainList.find(chain => chain.id === parseInt(id)) as Chain;
};
```

Right now devs would be limited to the chain ids in this map, but with `getChain` they can bring a Node for any chain that viem supports